### PR TITLE
button is updated

### PR DESCRIPTION
**Pull request by ayabaknu**
#### Background Information
changing button design

#### Impact
making all buttons have the same design 

#### Issue
Solves: https://github.com/team-refcode/refcode.org/issues/173

### CODE REVIEW

File: components/atoms/button.html [hunk 1]
```diff
@@ -1,21 +1,21 @@
-<div class="component" title="Button" version="1.1">
+<div class="component" title="Button" version="1.2">
   <includes>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet">
```
Comment (brentonstrine) [inline components/atoms/button.html:3]: You can delete the open sans include.

File: components/atoms/button.html [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend+Deca&display=swap" rel="stylesheet">
   </includes>
 
   <style>
-    .button {
```
Comment (brentonstrine) [inline components/atoms/button.html:7]: We actually need both `button` and `.button` here. You can use a comma to make a [selector list](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors#selector-list) so the rule applies to both.

File: components/atoms/button.html [hunk 3]
```diff
@@ -24,13 +24,9 @@
         text-align: center;
         cursor: pointer;
     }
-    a.button {
-        text-decoration: none;
-    }
```
Comment (brentonstrine) [inline components/atoms/button.html:29]: We still need the link button, so don't delete this one!

File: components/atoms/button.html [hunk 4]
```diff
@@ -24,13 +24,9 @@
...
   </style>
 
   <markup>
-    <a href="#" class="button">Link</a>
-    <button class="button">Button</button>
```
Comment (brentonstrine) [inline components/atoms/button.html:34]: Don't delete the link button! Also, keep he text generic for the component, like "Loren Ipsum" or "button"

**Review by brentonstrine — CHANGES_REQUESTED** [review]

**Comment by ayabaknu** [comment]
Ok. I will fix that now.

**Review by brentonstrine — APPROVED** [review]
Looks good!

**Pull request merged**